Gruntfile.js

```diff
@@ -0,0 +1,38 @@
+/*jslint es5: false */
+module.exports = function(grunt) {
+    'use strict';
+
+    //require('time-grunt')(grunt);
```

Comment:
I removed the comment tags. We actually do want to time grunt, so this needs to be included.